pkg/target/conduit/conduit.go

```diff
@@ -143,6 +150,7 @@ func (c *Conduit) Disconnect(ctx context.Context) error {
 	c.deleteVIPs(c.vips)
 	c.nexthops = []string{}
 	c.tableID = 1
+	c.Configuration.Delete()
```

Comment:
Since watching changes in vips are done via Configuration, wouldn't it be better Deleting the Configuration before cleaning up the vips?
On the other hand that cancel based stopping of WatchVIPs is asynchronous imho. So probably it is not granted that watch method would conclude before moving on to vip clean-up part...